legendary/cli.py

```diff
@@ -510,7 +513,38 @@ def launch_game(self, args, extra):
             logger.debug(f'Working directory: {cwd}')
             if env:
                 logger.debug('Environment overrides:', env)
-            subprocess.Popen(params, cwd=cwd, env=env)
+
+            manager = Manager()
+            ns = manager.Namespace()
+            ns.finish_rpc = False
+
+            if not args.disable_rpc:
+                rpc = Process(target=self.startRPC, args=(app_name, ns))
+                rpc.start()
+
+            game_process = subprocess.Popen(params, cwd=cwd, env=env)
+            game_process.wait()
+
+            if not args.disable_rpc:
+                ns.finish_rpc = True
+                rpc.join()
+
+
+    def startRPC(self, app_name, ns):
+        try:
+            RPC = Presence('828711025863688192')
```

Comment:
Everything looks good, but maybe add a comment explaining this magic number? I assume this is for connecting to Discord.

Reply:
I've now commented all of my code

Reply:
Yep, LGTM.